platform/ios/PolyPodApp/PolyPod/FeatureView.swift

```diff
@@ -89,20 +89,45 @@ struct FeatureView: View {
         .accessibilityIdentifier("feature_view")
     }
 
-    private func handleError(_ error: String) {
+    private func handleError(_ errorMsg: String) {
         let alert = UIAlertController(
             title: "",
             message: String.localizedStringWithFormat(
                 NSLocalizedString(
                     "message_feature_error %@ %@",
                     comment: ""
                 ),
-                feature.name, error
+                feature.name, errorMsg
             ),
             preferredStyle: .alert
         )
         alert.addAction(UIAlertAction(
-            title: "OK",
+            title: NSLocalizedString(
+                "button_error_report_allow",
+                comment: ""
+            ),
+            style: .default,
+            handler: { _ in
+                closeAction()
+                PodApi.shared.endpoint.uploadToServer(
+                    errorMsg: errorMsg,
+                    endpointId: "polyApiErrorReport",
+                    completionHandler: { error in
+                        if error != nil {
+                            Log.error("Upload failed: \(error!.localizedDescription)")
```

Comment:
You already know that error is not nil, right?

Reply:
in case of success it is `nil` indeed the returned value of the `completionHandler`